framework_crates/bones_framework/src/lib.rs

```diff
@@ -41,13 +47,21 @@ pub mod params;
 pub mod render;
 pub mod storage;
 pub mod time;
+pub mod utils;
+
+#[cfg(not(target_arch = "wasm32"))]
+pub mod networking;
 
 #[cfg(feature = "scripting")]
 pub use bones_scripting as scripting;
 
 #[cfg(feature = "localization")]
 pub mod localization;
 
+// TODO: make configurable
+/// Max players in networked game
+pub const MAX_PLAYERS: usize = 4;
```

Comment:
Do we need this constant here, when we have another `MAX_PLAYERS` in `networking.rs`.